riscv_private.h

```diff
@@ -210,3 +238,80 @@ static inline uint32_t sign_extend_b(uint32_t x)
 {
     return (int32_t)((int8_t) x);
 }
+
+// decode rs1 field
```

Comment:
Use `#ifdef ENABLE_RV32C` and `#end` for conditional build of these compressed instruction specific helper functions.